include/beast/core/handler_alloc.hpp

```diff
@@ -55,6 +55,12 @@ class handler_alloc
     using value_type = T;
     using is_always_equal = std::true_type;
 
+    template<class U>
+    struct rebind
+    {
+        using other = handler_alloc<U, CompletionHandler>;
+    };
+
```

Comment:
This looks right, however `std::allocator_traits` should have done this for you.  Did one of our std::libs whine about this?